torch/_tensor_str.py

```diff
@@ -327,7 +327,8 @@ def _str_intern(inp, *, tensor_contents=None):
     # In other cases, we don't have a way to set them as default yet,
     # and we should always print out device for them.
     if self.device.type != torch._C._get_default_device()\
-            or (self.device.type == 'cuda' and torch.cuda.current_device() != self.device.index):
+            or (self.device.type == 'cuda' and torch.cuda.current_device() != self.device.index)\
+            or (self.device.type == 'mps'):
```

Comment:
does MPS have a concept of default device? (could it, in the future?)

Reply:
There is a single MPSDevice which is the default. Currently we don't have support for multiple devices. So no concept of current_device.